.github/workflows/main.yml

```diff
@@ -19,6 +19,18 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           pip install -r requirements-dev.txt
+          curl https://github.com/ydirson/branding-xcp-ng/releases/download/untagged-630bba873b965a46c408/branding.py -O
```

Comment:
What is this? I assume this is temporary, Would be good to have a comment here.

Reply:
Yes, that's the whole reason this PR is in Draft state, will clarify

Reply:
This still returns a 404 :(

Reply:
If you need to host it somewhere temporarily might be better to just host it on gist.github.com, at least those URLs won't disappear on their own.

Reply:
That's really strange, I don't have any issue with this URL, even from a separate VM - I don't expect github to allow that one just because they share the same external IP. Maybe their infrastructure has inconsistencies between regions ?  Would sound a bit bad too, don't really know what to think...  Let's try a gist for now...

Reply:
Gist worked, thanks